app/views/juke/index.html.erb

```diff
@@ -1,46 +1,47 @@
-  <div class="site-wrapper-inner">
-    <div class="cover-container">
-      <div class="masthead clearfix">
-        <div class="inner">
-          <h3 class="masthead-brand">예수인교회 Jukebox</h3>
-          <nav>
-            <ul id='juke_control' class="nav masthead-nav">
-              <li>
-                <span data_url="/juke/previous" class="glyphicon glyphicon-backward" aria-hidden="true"></span>
-              </li>
-              <li id="play_button_container">
-                <%= render partial: 'play_button' %>
-              </li>
-              <li>
-                <span data_url="/juke/next" class="glyphicon glyphicon-forward" aria-hidden="true"></span>
-              </li>
-              <li>
-                <span data_url="/juke/stop" class="glyphicon glyphicon-stop" aria-hidden="true"></span>
-              </li>
-              <li>
-                <span data_url="/juke/clear" class="glyphicon glyphicon-eject" aria-hidden="true"></span>
-              </li>
-            </ul>
-          </nav>
-        </div>
-      </div>
+<section id="header" class="dark">
+	<h2 class="button scrolly" id="playlist">
+		<div class="inner cover" id="playing_now">
+			<%= render partial: 'playing_now' %>
+		</div>
+	</h2>
+	<header>
+		<h1 id="time">
+			<%= render partial: 'time' %>
+		</h1>
+	</header>
 
-      <div class="inner cover" id="playing_now">
-        <%= render partial: 'playing_now' %>
-      </div>
+	<footer>
+		<h1 class="button scrolly" style="margin-top:40px;">
+			<p class="controller" id="title"><%= render partial: 'title' %></p>
+			<ul id='juke_control' class="nav masthead-nav">
+	      <li>
+	        <span data_url="/juke/previous" class="glyphicon glyphicon-backward" aria-hidden="true"></span>
+	      </li>
+	      <li id="play_button_container">
+	        <%= render partial: 'play_button' %>
+	      </li>
+				<li>
+				  <span data_url="/juke/next" class="glyphicon glyphicon-forward" aria-hidden="true"></span>
+				</li>
+				<li>
+				  <span data_url="/juke/stop" class="glyphicon glyphicon-stop" aria-hidden="true"></span>
+				</li>
+				<li>
+				  <span data_url="/juke/clear" class="glyphicon glyphicon-eject" aria-hidden="true"></span>
+				</li>
+	    </ul>
+		</h1>
+	</footer>
+	<div class="mastfoot">
+		<div class="inner">
+			<p style="text-align:center; color:white;">FGSF Mediateam - Give us your feedback!<b> <a href="#" style="color:#91e7c1;">FGSFmedia@gmail.com</a></b></p>
+		</div>
+	</div>
+</section>
 
-      <div class="mastfoot">
-        <div class="inner">
-          <p>FGSF Media team - Give us your feedback! <a href=#">fgsfmedia@gmail.com</a></p>
-        </div>
-      </div>
-    </div>
-  </div>
-</div>
 
 <script>
-  setInterval(function() {
-    $('#playing_now').load('/juke/playing_now');
-  }, 2000);
-</script>
\ No newline at end of file
+setInterval(function() {$('#playing_now').load('/juke/playing_now');}, 2000);
+setInterval(function() {$('#time').load('/juke/time');}, 1000);
```

Comment:
tip (for future work): current time can be retrieved via jQuery  from http://stackoverflow.com/questions/20456712/how-to-get-current-time-with-jquery which saves us a server call.